.github/workflows/ci.yml

```diff
@@ -12,12 +12,60 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v1
-      - name: Set Xcode version to 11.6
-        run: sudo xcode-select -switch /Applications/Xcode_11.6.app
+        uses: actions/checkout@v2
+      - name: Set Xcode version to 11.7
+        run: sudo xcode-select -switch /Applications/Xcode_11.7.app
       - name: Build for testing
-        run:  xcodebuild  build-for-testing -scheme TABResourceLoader -configuration Debug -sdk iphonesimulator13.6 -destination "OS=13.6,name=iPhone 11 Pro"
+        run:  xcodebuild  build-for-testing -scheme TABResourceLoader -configuration Debug -sdk iphonesimulator13.7 -destination "OS=13.7,name=iPhone 11 Pro"
       - name: Test on iPhone 11 Pro
-        run:  xcodebuild test-without-building -scheme TABResourceLoader -configuration Debug -sdk iphonesimulator13.6 -destination "OS=13.6,name=iPhone 11 Pro" -enableCodeCoverage YES
+        run:  xcodebuild test-without-building -scheme TABResourceLoader -configuration Debug -sdk iphonesimulator13.7 -destination "OS=13.7,name=iPhone 11 Pro" -enableCodeCoverage YES
       - name: codecov
-        run: bash <(curl -s https://codecov.io/bash)    
+        run: bash <(curl -s https://codecov.io/bash)
+        
```

Comment:
In addition that could be great. If the test failed. It will be great to upload as an artefact the xcresult so it makes it easy to download the failure result and see on the logs in detail on why the tests are failing.

```yml
- name: Archive tests results
  if: ${{ failure() }}
  uses: actions/upload-artifact@v2
  with:
   name: Test-TABResourceLoader-Xcode12.1-iOS14.xcresult
   path: /Users/runner/Library/Developer/Xcode/DerivedData/*/Logs/Test/*.xcresult
```

Reply:
Good point, will add coverage 👍